pkg/controller/backup/backup_controller.go

```diff
@@ -156,6 +158,32 @@ func (c *Controller) updateBackup(cur interface{}) {
 		return
 	}
 
+	if v1alpha1.IsBackupLastRunning(newBackup) || v1alpha1.IsBackupLastPrepare(newBackup) {
```

Comment:
1. I think we should also check `Failed` beofore this block, and log and return if the CR already failed.
2. We cannot check the `Last` condition to determine the current status because there are two controllers, the backup controller in tidb-controller-manager and the backup controller in the backup manager, and they update the status separately, so the last condition maybe `Scheduled` or others, we can just check the condition as we're doing for `IsBackupComplete`.
3. We should also consider the case that `Prepare` condition is not updated before killed, so we should also check `Scheduled` here? And if no Pod created, will `err != nil ` in L163?